website/docs/reference/warehouse-setups/trino-setup.md

```diff
@@ -48,33 +48,34 @@ pip is the easiest way to install the adapter:
 
 <p>For further info, refer to the GitHub repository: <a href={`https://github.com/${frontMatter.meta.github_repo}`}>{frontMatter.meta.github_repo}</a></p>
 
-## Connecting to Starburst/Trino with dbt-core
+## Connecting to Starburst/Trino
 
-With dbt-core, the way to connect to your data platform is to creating a `profile` and `target` within the user-configured `profiles.yml` in the `.dbt/` directory of your user/home directory. For more information, please see both [Connection profiles](connection-profiles) and the [profiles.yml](../profiles.yml.md) reference page.
+To connect to a data platform with dbt Core, create appropriate _profile_ and _target_ YAML keys/values in the `profiles.yml` configuration file for your Starburst/Trino clusters. This dbt YAML file lives in the  `.dbt/` directory of your user/home directory. For more information, refer to [Connection profiles](/docs/core/connection-profiles) and [profiles.yml](/reference/profiles.yml). 
 
-The available profile parameters within dbt-trino can be broken into the following three categories:
+The parameters for setting up a connection are for Starburst Enterprise, Starburst Galaxy, and Trino clusters. Unless specified, "cluster" will mean any of these products' clusters.
 
-- host-related
-- authentication-related, and
-- dbt-specific- and session-related
+## Host parameters
 
-the below sections go into greater detail
+The following profile fields are always required except for `user`, which is also required unless you're using the `oauth`, `cert`, or `jwt` authentication methods.
 
-### Host parameters
+| Field     | Example | Description |
+| --------- | ------- | ----------- |
+|   `host`   | `mycluster.mydomain.com` | The hostname of your cluster.<br/><br/>Don't include the `http://` or `https://` prefix.  |
+| `database` | `my_postgres_catalog` | The name of a catalog in your cluster. |
+|  `schema`  | `my_schema`  | The name of a schema within your cluster's catalog. <br/><br/>It's _not recommended_ to use schema names that have upper case or mixed case letters.  |
```

Comment:
@dataders : quick question, why is upper/mixed case letters not recommended? do people encounter problems if they do?